src/components/BacktestOptionsPanel/BacktestOptionsPanel.js

```diff
@@ -110,60 +136,59 @@ const BacktestOptionsPanel = ({
           <p className='hfui-orderform__input-label'>{t('strategyEditor.useMarginCheckboxDescription')}</p>
         </div>
       )} */}
-      <div className='hfui-strategy-options__amount-input item'>
+      <div className='item'>
         <Checkbox
           label={t('strategyEditor.useTradesCheckbox')}
           checked={trades}
           onChange={setTrades}
+          disabled={isLoading}
         />
-        <div className='hfui-orderform__input-label'>
-          <Truncate>
-            {t('strategyEditor.useTradesCheckboxDescription')}
-          </Truncate>
+        <div className={clsx('hfui-orderform__input-label', { disabled: isLoading })}>
+          {t('strategyEditor.useTradesCheckboxDescription')}
         </div>
       </div>
-      <div className='hfui-strategy-options__input item'>
-        <DateInput
-          onChange={setStartDate}
-          def={{ label: t('strategyEditor.startDate') }}
-          value={startDate}
-          maxDate={endDate}
-        />
-      </div>
-      <div className='hfui-strategy-options__input item'>
-        <DateInput
-          onChange={setEndDate}
-          def={{ label: t('strategyEditor.endDate') }}
-          value={endDate}
-          maxDate={MAX_DATE}
-          minDate={startDate}
-        />
+
+      {isLoading && (
+      <div className='item'>
+        <Button
+          intent={Intent.NONE}
+          onClick={onCancelProcess}
+          className='hfui-strategy-backtest-options__start-btn'
+        >
+          {t('strategyEditor.cancelThisProcess')}
+        </Button>
+
       </div>
-      <div className='hfui-strategy-options__buttons-container item'>
-        {isFinished && (
+      )}
+      <div className='item'>
+        {isLoading ? (
           <Button
-            className='hfui-strategy-options__option-btn item'
-            label={t('strategyEditor.fullscreenChartBtn')}
-            onClick={setFullScreenChart}
-            green
-          />
+            className='hfui-strategy-backtest-options__start-btn'
+            intent={Intent.INFO}
```

Comment:
lets not have blue color for disabled state, either keep green with diff shade for disabled state or white/gray color